druid-pig/src/main/resources/META-INF/services/io.druid.initialization.DruidModule

```diff
@@ -0,0 +1 @@
+io.druid.guice.ServerModule
```

Comment:
external module facility is really to support extension loading. since this is internal, can you pls update DruidInitialization.java directly to have it with the iterable that is used to create the injector.